docs/guide/mocking.md

```diff
@@ -343,6 +343,11 @@ const mock = vi.fn(() => console.log('executed'))
 
 describe('delayed execution', () => {
   beforeEach(() => {
+    // Note: callng useFakeTimers can causes async function to hang
```

Comment:
Fake timers cannot cause async functions to hang. Only if the function uses a timer-based resolve.

Reply:
Out of topic: anyway, I have given up using mock dates, as it doesn't play well with integration test, for example Google Auth failed after the date object is mocked.